common/source/docs/common-smart-battery-landingpage.rst

```diff
@@ -0,0 +1,45 @@
+.. _common-smartbattery-landingpage:
+
+=============================================
+Smart Batteries
+=============================================
+
+ArduPilot supports three types of "smart batteries" that communicate using `SMBus <https://en.wikipedia.org/wiki/System_Management_Bus>`__: the Maxell battery, the `Solo battery <https://www.amazon.com/3DR-BT11A-Solo-Smart-Battery/dp/B00X97EXEU>`__, and the `Rotoye BatMon <https://rotoye.com/>`__.
+While not yet very common, smart batteries are easier to attach and detach from the vehicle and are capable of providing more information on the state of the battery including capacity, individual cell voltages, temperature, etc.
+
+.. note::
+
+   Rotoye BatMon is supported in Copter-v4.1.0 dev, but not yet in a stable release. Support for Solo smart batteries was introduced in Copter-3.3, Maxell battery support arrived in Copter-3.5.
+   We have not yet found a webstore that sells the Maxell smart batteries.
```

Comment:
as to the note re Maxell battery, I'm a bit conflicted...why mention a battery that cannot be obtained? (how did it get into the code if not available? part of a larger system that is?) or if we do, we should remove the availability note since it can change and become a maintenance issue for the wiki...I am fine with either approach

Reply:
ok. will remove that